### PR TITLE
FIXES :- Improve review feature documentation #3361

### PR DESCRIPTION
**Pull request by Rishavgupta12345**
added the required detailed information in the review section under #3361 

This pull request is created to verify the changes and discuss them with the mentors , and then finalise the commit

### CODE REVIEW

**Comment by 4D17Y4** [comment]
This is certainly a good idea, but i would prefer info icons near questions.

**Comment by Rishavgupta12345** [comment]
@nicolas-raoul

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -455,7 +455,7 @@ Upload your first media by tapping on the add button.</string>
...
 
-  <string name="review_image_explanation">You can review images and improve the quality of Wikimedia Commons.\n The four parameters of review are: \n - Is this image in-scope? \n - Does this image follow the rules of copyright? \n - Is this image correctly categorized? \n - If all goes well you can also thank the contributor.</string>
+  <string name="review_image_explanation">You can review images and improve the quality of Wikimedia Commons.\n The four parameters of review are: \n\n - Is this image in-scope? \n     when you say no out of scope, you are adding a  nomination for deletion template to this image. \n\n - Does this image follow the rules of copyright? \n     when you say no does not follow the rules of copyright, you are adding a  nomination for deletion template to this image. \n\n - Is this image correctly categorized? \n     when you say no correctly not categorized, you are adding a  nomination for deletion template to this image. \n\n - If all goes well the image is not nominated for further deletion and you can also thank the contributor.</string>
```
Comment (nicolas-raoul) [inline app/src/main/res/values/strings.xml:458]: Could you please replace with this?
```
You can review images to improve the quality of Wikimedia Commons.\nThe four parameters of review are:\n\n- Is this image in-scope?\nWhen you tap "No" (out of scope), you are adding a deletion nomination template to this image.\n\n- Does this image follow the rules of copyright?\nWhen you tap "No" (does not follow the rules of copyright), you are adding a deletion nomination template to this image.\n\n- Is this image correctly categorized?\nWhen you tap "No" (not correctly categorized), you are adding a categorization request template to this image.\n\nIf everything is fine, no template is added to the image, and you get the opportunity thank the contributor.
```
Thanks!

**Review by nicolas-raoul — CHANGES_REQUESTED** [review]

**Review by Rishavgupta12345 — COMMENTED** [review]
done the requested changes @nicolas-raoul

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -455,7 +455,7 @@ Upload your first media by tapping on the add button.</string>
...
   <string name="skip_image_explanation">Clicking this button will give you another recently uploaded image from Wikimedia Commons</string>
 
-  <string name="review_image_explanation">You can review images and improve the quality of Wikimedia Commons.\n The four parameters of review are: \n - Is this image in-scope? \n - Does this image follow the rules of copyright? \n - Is this image correctly categorized? \n - If all goes well you can also thank the contributor.</string>
+  <string name="review_image_explanation">You can review images to improve the quality of Wikimedia Commons.\nThe four parameters of review are:\n\n- Is this image in-scope?\nWhen you tap "No" (out of scope), you are adding a deletion nomination template to this image.\n\n- Does this image follow the rules of copyright?\nWhen you tap "No" (does not follow the rules of copyright), you are adding a deletion nomination template to this image.\n\n- Is this image correctly categorized?\nWhen you tap "No" (not correctly categorized), you are adding a categorization request template to this image.\n\nIf everything is fine, no template is added to the image, and you get the opportunity thank the contributor.</string>
```
[the remark this answers is not in the document]
Comment (nicolas-raoul) [reply app/src/main/res/values/strings.xml:458]: Also, I just realized that quotes do not show up and break the XML, so could you please replace all of the `"No"` with `No`? Thanks!

**Review by Rishavgupta12345 — COMMENTED** [review]
done @nicolas-raoul

File: app/src/main/res/values/strings.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Rishavgupta12345) [reply app/src/main/res/values/strings.xml]: done

**Comment by nicolas-raoul** [comment]
Thanks!

**Review by nicolas-raoul — APPROVED** [review]

**Pull request merged**